README.md

```diff
@@ -0,0 +1,47 @@
+# CRR Training - Buiding a Mobile Menu for a Portfolio
+
+> Building a mobile menu for a portfolio website use HTML, CSS and Vanilla Javascript
+
+## Live Demo
```

Comment:
It is commendable that you deployed your project and shared the link to your deployed version. :clap: However, I think your README.md file would make your project easier to navigate if you added a screenshot of your live project. The `Contributing`, `Show your support`, and `Acknowledgements` sections would make it easy for other developers to contribute to your project. You may use [this template](https://github.com/microverseinc/readme-template).